dbtools/pg.py

```diff
@@ -422,6 +422,33 @@ def get_engine_pools(self, pool_size=5, max_overflow=10):
 
         return engine
 
+    def get_pg_relkind(self, relname: str, schema: str) -> Literal['r', 'i', 'S', 'v', 'm', 'c', 't', 'f']:
+        """Get the type of object that relname is.
+
+        r = ordinary table
+        i = index
+        S = sequence
+        v = view
+        m = materialized view
+        c = composite type
+        t = TOAST table
+        f = foreign table

```

Comment:
Could potentially write this into an `Enum`, then check `results` to ensure they are members of the `Enum`.

Reply:
as discussed - nah